packages/react-components/react-card/src/components/Card/useCardStyles.ts

```diff
@@ -38,6 +39,29 @@ const useStyles = makeStyles({
     ...shorthands.padding(`var(${cardCSSVars.cardSizeVar})`),
     ...shorthands.gap(`var(${cardCSSVars.cardSizeVar})`),
 
+    [`> .${cardHeaderClassNames.root}, > .${cardFooterClassNames.root}`]: {
+      flexShrink: 0,
+    },
+    [`> :not(.${cardPreviewClassNames.root}):not(.${cardHeaderClassNames.root}):not(.${cardFooterClassNames.root})`]: {
+      flexGrow: 1,
+    },
+  },
+
+  orientationHorizontal: {
+    flexDirection: 'row',
+    alignItems: 'center',
+
+    [`> .${cardPreviewClassNames.root}`]: {
+      marginTop: `calc(var(${cardCSSVars.cardSizeVar}) * -1)`,
+      marginBottom: `calc(var(${cardCSSVars.cardSizeVar}) * -1)`,
+    },
+    [`> :not([aria-hidden="true"]):first-of-type.${cardPreviewClassNames.root}`]: {
```

Comment:
Unfortunate necessity due to Tabster's injected elements

Reply:
Im usually against adding comments to codebase, but I think this is the exception. can we document this pls?

Reply:
I think that's warranted 😓 Will do 👍🏻

Reply:
I went ahead and added a little explanation to the other styles too. Let me know if you would change something.

Reply:
all good lets get this in!

Reply:
thanks for reaching out @spmonahan ! The thing with Tabster DOM insertions is it can break at any time (which happened already). IMO would be best if tabster could expose a hook (css selector?) as part of its public API to give use solid contract. Can we reach out  to folks @andrefcdias  ? WDYT

Reply:
Thank you for sharing your concerns, Sean. I agree that this approach is not ideal, but please keep in mind that there will be some work put into optimization after all implementation is finished as not to keep delaying the implementation.
As Martin said, Tabster updates tends to be breaking and this was the lesser evil approach.
I'll keep this in mind for the future before finalizing Card.